data/nav_graphql.yml

```diff
@@ -9,8 +9,6 @@
     path: apis/graphql/graphql-cookbook
   - name: Agents
     path: apis/graphql/cookbooks/agents
-  - name: Artifacts
-    path: apis/graphql/cookbooks/artifacts
```

Comment:
Actually, this was a new section added as part of https://github.com/buildkite/docs/pull/2757.

Why is the GraphQL schema bot job attempting to removing this section of our docs?

Reply:
Ah right, `nav_graphql` is generated from a script. 

The following code will need to be updated with the new cookbook section.

https://github.com/buildkite/docs/blob/main/scripts/graphql_api_content/nav_data.rb#L17-L105

Reply:
That file should probably have a comment indicating that it was generated. I can add something for that.

Reply:
OK cool, I've popped up a PR to make this a little more obvious.

https://github.com/buildkite/docs/pull/2779